docker/ansible/viame-deploy.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Run the Viame deploy every Sunday and Thursday at 2 AM
+
+[Timer]
+OnCalendar=Sun 02:00
+OnCalendar=Thu 02:00
```

Comment:
This relies on the system clock, so these times are relative to the system time zone.    Should probably specify a time zone here.  

`Sun 02:00 UTC-5` would be EST, I believe, as would `Sun 02:00 America/New_York`

Reply:
I agree

Reply:
edd6ce2